internal/engine/buildcontroller_test.go

```diff
@@ -47,15 +46,17 @@ func TestBuildControllerTwoPods(t *testing.T) {
 	assert.Equal(t, manifest.ImageTargetAt(0), call.image())
 	assert.Equal(t, []string{}, call.oneState().FilesChanged())
 
+	// Associate the pods with the manifest state
 	podA := f.testPod("pod-a", "fe", "Running", testContainer, time.Now())
 	podB := f.testPod("pod-b", "fe", "Running", testContainer, time.Now())
-	image := fmt.Sprintf("%s:%s", f.imageNameForManifest("fe").String(), "now")
```

Comment:
this code is critical for the test, because it makes sure both pods have the same image. (otherwise, they will get different image ids)

Reply:
But we no longer match on the image the pod is running, so this is moot, right? Is there some other behavior you think we should be testing?

Reply:
(`testPod` be default labels the pods with TestDeployId, and TestBuildAndDeployer by default deploys with TestDeployId, so BOTH these pods will get attached to `manifest`.)